homeassistant/components/stream/worker.py

```diff
@@ -44,13 +44,10 @@ def __init__(self, outputs_callback) -> None:
 
     @staticmethod
     def make_new_av(
-        sequence: int, input_vstream: av.video.VideoStream, input_astream: Any
-    ) -> tuple[
-        BytesIO, av.container.OutputContainer, av.video.VideoStream, Any
-    ]:  # fix audio stream types later
-        """Reset the av objects for this segment."""
-        memory_file = BytesIO()
-        av_output = av.open(
+        memory_file, sequence: int, input_vstream: av.video.VideoStream
```

Comment:
Optional: vstream could be a time base.

Reply:
No preference on that itself. But the question is if we go to time base, why not pass in the reciprocal of the time_base? But that might become harder to read

Reply:
Also if we did that we might change the `sequence` parameter to `fragment index` (ie add 1 in reset instead of in make_new_av)
Do you think it's cleaner to have this translations in `make_new_av` or in `reset`?

Reply:
Interesting, I don't have strong opinions on these. I'm ok with leaving any of the 3.

Reply:
Ok, how about we just leave it as is for now. It will keep the call from `reset` a little cleaner